### DISCUSSION TITLE
InAppBillingPlugin support for NET8

### DISCUSSION THREAD

**Discussion by hflexgrig** [dotnet/maui#24258] [Q&A] [answered]
I just recently upgraded my app from Xamarin to MAUI, and urgently need [InAppBillingPlugin](https://github.com/jamesmontemagno/InAppBillingPlugin) to be operational. As I see it has dependency on https://github.com/xamarin/GooglePlayServicesComponents/pull/815

Is there any workaround I can get it working asap on my app?

**Comment by phillippschmedt** [comment, marked as answer]
We are using the plugin in version 7.1.3 with .NET 8 on iOS and Android and it works just fine. See discussion here: https://github.com/jamesmontemagno/InAppBillingPlugin/issues/623
